templates/shared/header.html

```diff
@@ -35,13 +35,13 @@
             </a>
         </li>
         <li class="p-navigation__link" role="menuitem">
-          <a class="p-link--external" href="#">Discourse</a>
+          <a class="p-link--external" href="https://discourse.jujucharms.com/">Discourse</a>
         </li>
         <li class="p-navigation__link" role="menuitem">
-          <a class="p-link--external" href="#">Docs</a>
+          <a class="p-link--external" href="https://docs.jujucharms.com/">Docs</a>
         </li>
         <li class="p-navigation__link" role="menuitem">
-          <a class="p-link--external" href="#">Get started</a>
+          <a class="p-link--external" href="https://jujucharms.com/new/">Get started</a>
```

Comment:
You might as well change this to "My models" now

Reply:
I think we said “Your models”